x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v2_windows_rare_metadata_user.json

```diff
@@ -0,0 +1,37 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "This is a new refactored job which works on ECS compatible events across multiple indices. Security: Windows - Looks for anomalous access to the metadata service by an unusual user. The metadata service may be targeted in order to harvest credentials or user data scripts containing secrets.",
+  "groups": [
+    "security",
+    "endpoint",
+    "event-log",
+    "process",
+    "sysmon",
+    "windows",
+    "winlogbeat"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"user.name\"",
+        "function": "rare",
+        "by_field_name": "user.name"
+      }
+    ],
+    "influencers": [
```

Comment:
@blaklaybul should we add the `by` field `user.name` to the list of influencers?

Reply:
I will re-test with these proposed influencers and evaluate.

Reply:
yes, we should add this. We should mimic the existing job config here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/server/models/data_recognizer/modules/siem_winlogbeat/ml/windows_rare_metadata_user.json